helper/src/test/java/com/dailylife/helper/tasks/TaskRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package com.dailylife.helper.tasks;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskRepositoryTest {
+
+
+    @Test
+    public void testRepositoryWithOneElement(){
+
+        TaskRepository taskRepository = new TaskRepository();
+        Task task = new Task();
+
+        taskRepository.save(task);
+        Collection<Task> tasks = taskRepository.findAll();
+
+        assertEquals(1, tasks.size());
+        assertEquals(task,tasks.toArray()[0]);
+    }
+
+    @Test
+    public void testEmptyRepository(){
+        TaskRepository taskRepository = new TaskRepository();
+        Collection<Task> tasks = taskRepository.findAll();
```

Comment:
be consistent with vertical spacing here as well, if in other tests you have blank line before assertions, make the same here